integration/simple_app_test.go

```diff
@@ -75,10 +123,34 @@ func testSimpleApp(t *testing.T, context spec.G, it spec.S) {
 			Eventually(container).Should(BeAvailable())
 			Eventually(container).Should(Serve(ContainSubstring("Hello world!")).OnPort(9292))
 
-			logs, err := docker.Container.Logs.Execute(container.ID)
+			logs, err = docker.Container.Logs.Execute(container.ID)
 			Expect(err).NotTo(HaveOccurred())
-			Expect(logs.String()).To(ContainSubstring("retry"))
-			Expect(logs.String()).To(ContainSubstring("Set for the current user (/layers/paketo-buildpacks_bundle-install/gems/config): 5"))
+			Expect(logs).To(ContainLines(
+				"retry",
+				"Set for the current user (/layers/paketo-buildpacks_bundle-install/launch-gems/config): 5",
+				"",
+				"clean",
+				`Set for the current user (/layers/paketo-buildpacks_bundle-install/launch-gems/config): "true"`,
+				"",
+				"path",
+				`Set for the current user (/layers/paketo-buildpacks_bundle-install/launch-gems/config): "/layers/paketo-buildpacks_bundle-install/launch-gems"`,
+				"",
+				"without",
+				"Set for the current user (/layers/paketo-buildpacks_bundle-install/launch-gems/config): [:development, :test]",
+				"",
+				"user_config",
+				`Set via BUNDLE_USER_CONFIG: "/layers/paketo-buildpacks_bundle-install/launch-gems/config"`,
+			))
+
+			Expect(logs).To(ContainLines(
```

Comment:
Would it make sense to add a `NotTo(ContainLines())` for the development/test gems just to make sure they're not getting logged?

Reply:
Sure. I can do this.